.gitignore

```diff
@@ -17,3 +17,4 @@ pkg/
 /tmp/
 /yarn-error.log
 /test-reports/
+.idea
```

Comment:
This is specific to your environment. Would you mind removing this and using a global gitignore for these kinds of files?

Reply:
Yeah, this should be removed.

Reply:
sure, i will remove